### DISCUSSION TITLE
v1.14.1

### DISCUSSION THREAD

**Discussion by slawekjaranowski** [s4u/pgpverify-maven-plugin#308] [Announcements]
# What's Changed

## Bug Fixes

- propagate KeyNotFound on first server to client #307

## Dependency updates

- Bump jackson.version from 2.12.3 to 2.12.4 #302
- Bump commons-io from 2.10.0 to 2.11.0 #303

# Thanks
Many thanks for collaboration on this release for:   @slawekjaranowski, @aoindustries

<hr /><em>This discussion was created from the release <a href='https://github.com/s4u/pgpverify-maven-plugin/releases/tag/v1.14.1'>v1.14.1</a>.</em>

**Comment by ao-apps** [comment]
Thank you.
